.travis.yml

```diff
@@ -4,6 +4,11 @@ notifications:
   email: false
 install:
   - pip install -r test_requirements.txt
+before_script:
+  - git clone https://github.com/sphinx-doc/sphinx ../sphinx
+  - (cd .. && ln -s sphinx/sphinx/templates/quickstart/ sphinx-quickstart-templates)
+  # Check upstream mobans are in sync
+  - moban -m upstream.yml
```

Comment:
I think this command will update sphinx related files in the repo and then the tests will be performed on top of them. Do you imagine that we update them offline? for example, when the build fails, and we try to update those sphinx files, e.g. Make.bat.jj2. How would the update work?

Reply:
I believe make.bat should not be templated and must be autogenerated. Refer [this](http://www.sphinx-doc.org/en/master/usage/quickstart.html#running-the-build). I believe changes must be done in `setup.py` to autogenerate this.